modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePriceLists.testcase

```diff
@@ -2001,6 +2004,7 @@ definition {
 			priceListName = "Test Price List");
 	}
 
+	@priority = 3
 	test PriceListExistsOnlyInOneCatalog {
 		property portal.acceptance = "false";
 		property portal.upstream = "true";
```

Comment:
Can you remove this line from these tests?

> property portal.upstream = "true";

This is not needed as the property is set to true for the entire file, thanks!